src/appleseed.shaders/src/maya/as_maya_place2dTexture.osl

```diff
@@ -157,12 +157,7 @@ shader as_maya_place2dTexture
     ]]
 )
 {
-    // The user might override input UV coords for the placement node.
```

Comment:
@est77 my only concern was piping outUv from a node to the input UV of the placement node (it's possible), but if a connection is set, it should automagically be picked up with fallback as global primvars u and v, so everything ok.